ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -227,11 +225,12 @@ QObject {
     }
 
     /* Internal function to calculate total balance */
-    function __getTotalBalance(balances, decimals) {
+    function __getTotalBalance(balances, decimals, chainIds = [root.swapFormData.selectedNetworkChainId]) {
```

Comment:
ideally there must be another way to calculate total balance now? using the SubModelProxyModel ?

Reply:
Yeah, same comment as above, I'll get rid of this completely